### PR TITLE
patch attribution href in footer to correct path

### PR DESCRIPTION
**Pull request by kovaca**
## Overview

_Please write a description. If the PR is hard to understand, provide a quick explanation of the code._

## Testing instructions
 * How to test this PR
 * Prefer bulleted description
 * Start after checking out this branch
 * Include any setup required, such as bundling scripts, restarting services, etc.
 * Include test case, and expected output

## Jira task / Github issue
_Provide the link to the Jira task(s), if any._

Resolves #XXX

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Checklist before submitting
- [ ] Title: Don't forget to make it clear for everyone, even for people that don't know about the feature/bug.
- [ ] Meaningful commits and code rebased on `develop`.
- [ ] Add entry to CHANGELOG.md, if appropriate

### CODE REVIEW

**Comment by andresgnlez** [comment]
Hey Alex, thanks for the contribution! I am afraid the change would break the link in the next deployment to production as with the changes performed in the last months the route work as [https://preproduction.resourcewatch.org/api-attribution-requirements](https://preproduction.resourcewatch.org/api-attribution-requirements) right now if preproduction, which has latest changes.

TLDR; this will be automatically fixed with the next deployment to production scheduled by this week. If by any chance you want to rename the route to what you proposed is not a problem but I would need to do an additional change on my side, so let me know :)

**Comment by kovaca** [comment]
Hi Andrés, ok that's great and thanks for catching it. Let's stick with the fix you've already made to routes. I'll close this.
